### PR TITLE
improve backup checksum feature

### PR DESCRIPTION
**Pull request by NingLin-P**
Signed-off-by: linning <linningde25@gmail.com>
improve backup checksum feature

### CODE REVIEW

File: proto/backup.proto [hunk 1]
```diff
@@ -39,17 +39,25 @@ message BackupMeta {
 
 message File {
     string name = 1;
-    uint32 crc32 = 2;
+    uint64 content_checksum = 2;
```
Comment (overvenus) [inline proto/backup.proto]: Could you give it a self-described name?

Comment (3pointer) [inline proto/backup.proto]: shoud we change it to `string`, if we use hash function like [sha-1](https://docs.rs/rust-crypto/0.2.36/crypto/sha1/index.html)

Comment (NingLin-P) [reply proto/backup.proto]: like the used algorithm name?

Comment (5kbpers) [reply proto/backup.proto]: prefer to use `bytes`

Comment (overvenus) [reply proto/backup.proto]: Or maybe `bytes`?

Comment (NingLin-P) [reply proto/backup.proto]: OK, I will use `bytes`

**Pull request merged**